src/Config.php

```diff
@@ -1233,7 +1250,21 @@ public function processLongArgument($arg, $pos)
                     break;
                 }
 
-                $this->generator = substr($arg, 10);
+                $generatorName          = substr($arg, 10);
+                $lowerCaseGeneratorName = strtolower($generatorName);
+
+                if (isset(self::$validGenerators[$lowerCaseGeneratorName]) === false) {
+                    $validOptions = implode(', ', array_values(self::$validGenerators));
+                    $error        = sprintf(
+                        'ERROR: "%s" is not a valid generator. Valid options are: %s.'.PHP_EOL.PHP_EOL,
```

Comment:
Just a suggestion...

```suggestion
                        'ERROR: "%s" is not a valid generator. The following generators are supported: %s.'.PHP_EOL.PHP_EOL,
```